src/lib/viewers/media/Settings.js

```diff
@@ -183,7 +183,7 @@ class Settings extends EventEmitter {
         this.containerEl.classList.add(CLASS_SETTINGS_SUBTITLES_UNAVAILABLE);
         this.containerEl.classList.add(CLASS_SETTINGS_AUDIOTRACKS_UNAVAILABLE);
 
-        if (Browser.isIOS()) {
+        if (Browser.isMobile()) {
```

Comment:
Chrome doesn't allow play without a user action and setting the autoplay tag is not respected, so i've turned off autoplay for mobile in general. https://developers.google.com/web/updates/2016/07/autoplay